src/+io/loadModelSettings.m

```diff
@@ -0,0 +1,81 @@
+function ModelSettings = loadModelSettings()
```

Comment:
```suggestion
function ModelSettings = getModelSettings()
```
I'm not sure if "load" is the right name here: nothing is loaded from any files.